lib/po_libc_wrappers.c

```diff
@@ -129,36 +153,6 @@ stat(const char *path, struct stat *st)
 	return fstatat(rel.dirfd, rel.relative_path,st,AT_SYMLINK_NOFOLLOW);
 }
 
-struct po_map*
-po_map_get()
-{
-	if (global_map == NULL) {
-		global_map = po_map_create(4);
-	}
-
-	if (global_map != NULL) {
-		po_map_assertvalid(global_map);
-	}
-
-	global_map->refcount += 1;
-
-	return (global_map);
-}
-
-void
-po_map_set(struct po_map *map)
```

Comment:
Checking out your pull request, I see that the tests now fail due to a lack of `po_map_set()`... I've just pushed a new commit that does some of the same simplification but preserves a (new) `po_set_libc_map()` function in the internal header file, just for the sake of tests. Could you rebase your work on this new version of `origin/master`?